packages/gatsby-plugin-google-analytics/src/gatsby-ssr.js

```diff
@@ -1,5 +1,19 @@
 import React from "react"
 
+const knownOptions = {
```

Comment:
@mattwexa you forgot about `name` and `cookieDomain` options

Reply:
@tradziej yes, this is by design. `cookieDomain` and `name` are treated differently than other create only fields: if they are specified, they are given as parameters to `ga('create')` directly rather than in the gaCreateOptions object.

Reply:
@mattwexa sure, thank you for explanation